gluster.yml

```diff
@@ -5,8 +5,9 @@
 - hosts: gluster
   remote_user: root
   vars:
-    install_from: packages
+    gluster_centos_repo: "nightly"
+    # gluster_centos_repo: "3-11" # for GlusterFS 3.11
   roles:
     - gluster-centos-repo
-    - gluster-gdeploy-copr
+    # - gluster-gdeploy-copr
```

Comment:
If you don't want to use this role then it would be better to remove it instead of commenting it out.

Reply:
I didn't remove it here to highlight that one can still use this role here (in a scenario which is no longer a primary one, but it should do something in upstream).